README.md

```diff
@@ -198,12 +212,12 @@ npm run start
 
 ## Continuous Deployment
 
-We have set up Travis to deploy Agora to our [AWS infrastructure](https://github.com/Sage-Bionetworks/Agora-infra).
+We have set up Travis to deploy Agora to our [AWS infrastructure](https://github.com/Sage-Bionetworks/agora2-infra).
```

Comment:
The previous link doesn't work, so updating to (I think) the current infra repo

Reply:
nice catch, yes that is the correct infra repo for our new(ish) environments